### DISCUSSION TITLE
Cannot build project with webpack and aws-sdk/client-ses

### DISCUSSION THREAD

**Discussion by harshasri77** [webpack/webpack#17128] [Q&A]
Hi all,

   I have bundled a package with @aws-sdk/client-ses version 3.325.0.  ------> this one will be my custom library. 
Trying to import that library in package.json into my service which uses webpack (version 5.75.0).
and trying to run npm run build

It is throwing an error as below

ERROR in ./node_modules/customlibraryname/src/ses/index.ts 4:45
Module parse failed: Unexpected token (4:45)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> export async function makeAWSCall(mailOptions: any, awsRegion: any) {
|   const sesClient = new SES({

**Comment by alexander-akait** [comment]
Please provide configuration, looks like you don't have `typesccript` files, do you have `ts-loader`?

**Reply by harshasri77** [reply]
yes I do have ts-loader. Below is my snippet of webpack.config.js file

  resolve: {
    extensions: ['.ts', '.js'],
    alias: {
      '@lib': path.join(__dirname, 'src/lib'),
    },
  },
  module: {
    rules: [
      {
        test: /\.ts$/,
        exclude: /node_modules/,
        use: [
          {
            loader: 'ts-loader',
            options: {
              transpileOnly: true
            }
          }
        ]
      }
    ]
  },
  plugins: [
    new ForkTsCheckerWebpackPlugin({
      async: false,
      issue: {
        include: [
          { file: '**/src/**/*' }
        ],
        exclude: [
          { origin: 'eslint', file: '**/*.spec.ts' }
        ]
      },
      typescript:{
        enabled:true,
        configFile: './tsconfig.json',
        memoryLimit: 10096,
      }      
    })
  ]
}

**Reply by alexander-akait** [reply]
You have this `exclude: /node_modules/,`, but your file is here `./node_modules/customlibraryname/src/ses/index.ts 4:45`, just remove this line

**Reply by harshasri77** [reply]
I have remove that line. Now I am getting two new below errors.

1) WARNING in ./node_modules/@aws-sdk/util-user-agent-node/dist-es/is-crt-available.js 3:78-96
Module not found: Error: Can't resolve 'aws-crt' in '\node_modules\@aws-sdk\util-user-agent-node\dist-es'

2)WARNING in ./node_modules/dtrace-provider/dtrace-provider.js 18:22-85
Module not found: Error: Can't resolve './src/build' in '\node_modules\dtrace-provider'

**Reply by alexander-akait** [reply]
Sounds like problems on a library side, do you really need to build `typescript` version of `@aws-sdk`?
